var/spack/repos/builtin/packages/hdf5-vol-async/package.py

```diff
@@ -27,7 +27,6 @@ class Hdf5VolAsync(CMakePackage):
     version("1.6", tag="v1.6")
     version("1.5", tag="v1.5")
     version("1.4", tag="v1.4")
-    version("1.3", tag="v1.3")
     version("1.2", tag="v1.2")
```

Comment:
The dependencies suggest all ofthe versions older than 1.3 are not compatible with HDF5 1.14

Reply:
I removed the older versions in #38026

Reply:
great, approved that one.